packages/core/admin/admin/src/utils/strings.ts

```diff
@@ -1,3 +1,38 @@
 const capitalise = (str: string): string => str.charAt(0).toUpperCase() + str.slice(1);
 
-export { capitalise };
+function getByteSize(value: string) {
+  if (typeof TextEncoder !== 'undefined') {
```

Comment:
When would `TextEncoder` ever be undefined? Is this just for the testing environment?

Reply:
AFAIK the test environment or a very old browser

Reply:
I see a lot of suggestions for [providing the implementation](https://github.com/inrupt/solid-client-authn-js/issues/1676#issuecomment-917016646) to the jest environment. Curious if you tried that? I'm not sure the manual implementation below is worth it to support such outdated browsers. Could just be me and my little brain but I struggle to understand what's going on down there 🤯

Reply:
I'll try that and see!

Reply:
Ok, seems to work, we can go with that instead.